lua/sim/SelectSpawnLocation.lua

```
@@ -0,0 +1,520 @@
+end
+
+function RenderMarkers()
+
+    local tblArmy = ListArmies()
+    local focusArmy = GetFocusArmy()
+
+    Sync.Markers = {}
+    for iArmy, strArmy in pairs(tblArmy) do
+        local armyIsCiv = ScenarioInfo.ArmySetup[strArmy].Civilian
+
+        if armyIsCiv then continue end
```

Luacheck (1.2.0)

[error] expected '=' near 'end'

(E011)